flex-backend/main.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 func main() {
+	log.SetFlags(log.LstdFlags | log.Lshortfile)
```

Comment:
This will show the line numbers whenever we do a `log` statement. More helpful for debugging